specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,610 @@
+          }
+        }
+      },
+      "description": "An error response from the service."
+    },
+    "SqlRoleDefinitionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SqlRoleDefinitionGetResults"
+          },
+          "description": "List of Role Definitions and their properties."
+        }
+      },
+      "description": "The relevant Role Definitions."
+    },
+    "SqlRoleAssignmentCreateUpdateParameters": {
+      "type": "object",
+      "properties": {
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The unique identifier for the associated Role Definition."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The data plane resource path for which access is being granted through this Role Assignment."
+        },
+        "principalId": {
+          "type": "string",
+          "description": "The unique identifier for the associated AAD principal in the AAD graph to which access is being granted through this Role Assignment. Tenant ID for the principal is inferred using the tenant associated with the subscription."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Role Assignment Properties."
+    },
+    "SqlRoleAssignmentGetResults": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified GUID of the Role Assignment containing the resource path and the ID supplied for this Role Assignment during create."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID supplied for this Role Assignment during create."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SqlRoleAssignmentCreateUpdateParameters",
+          "description": "Properties related to the Role Assignment."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "An Azure Cosmos DB Role Assignment"
+    },
+    "SqlRoleAssignmentListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SqlRoleAssignmentGetResults"
+          },
+          "description": "List of Role Assignments and their properties"
+        }
+      },
+      "description": "The relevant Role Assignments."
+    }
+  },
+  "parameters": {
+    "roleDefinitionIdParameter": {
+      "name": "roleDefinitionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "The GUID for the Role Definition."
+    },
+    "accountNameParameter": {
```

Comment:
accountNameParameter should be re-used from cosmos-db.json

Reply:
Done